### PR TITLE
[doc] Add CM label for scrambling key sideload

### PR DESCRIPTION
**Pull request by msfschaffner**

### CODE REVIEW

**Review by rswarbrick — CHANGES_REQUESTED** [review]
Please could you update the comportability doc as well as the Python? (This could be squashed into the first commit)

**Review by GregAC — APPROVED** [review]
LGTM modulo @rswarbrick's comment

**Comment by msfschaffner** [comment]
> Please could you update the comportability doc as well as the Python? (This could be squashed into the first commit)

Ah yes - sorry about that. Completely forgot to add that in.

**Comment by msfschaffner** [comment]
@rswarbrick fixed!

File: doc/rm/comportability_specification/index.md [hunk 1]
```diff
@@ -299,6 +299,7 @@ The following standardised countermeasures are defined:
 | BKGN_CHK       | The asset is protected with a continuous background health check |
 | GLITCH_DETECT  | The asset is protected by an analog glitch detector | CTRL, FSM, CLK, RST
 | SW_UNREADABLE  | The asset is not accessible by software |
+| SIDELOAD       | The asset is can be loaded without exposing it to software | KEY
```
Comment (msfschaffner) [inline doc/rm/comportability_specification/index.md:302]: @vogelpi 
FYI: I took this from your AES CM label PR

Comment (rswarbrick) [reply doc/rm/comportability_specification/index.md:302]: I think you want to drop the "is" here?

**Review by rswarbrick — APPROVED** [review]
Other than a typo in the doc, this looks good to me!

**Comment by msfschaffner** [comment]
> Other than a typo in the doc, this looks good to me!

Woops - corrected!

File: doc/rm/comportability_specification/index.md (hunk 1 above)
Comment (vogelpi) [reply doc/rm/comportability_specification/index.md]: Sounds good, I will rebase the AES PR later today.

**Pull request merged**